metadata_service/proxy/atlas_proxy.py

```diff
@@ -706,6 +734,47 @@ def get_table_by_user_relation(self, *, user_email: str, relation_type: UserReso
                 cluster=res['cluster'],
                 schema=res['db'],
                 name=res['table']))
+        return results
+
+    def _get_resource_owned_by_user(self, user_id: str, resource_type: str) \
+            -> List[Union[PopularTable, DashboardSummary, Any]]:
+        """
+        ToDo (Verdan): Dashboard still needs to be implemented.
+        Helper function to get the resource, table, dashboard etc owned by a user.
+        :param user_id: User ID of a user
+        :param resource_type: Type of a resource that returns, could be table, dashboard etc.
+        :return: A list of PopularTable, DashboardSummary or any other resource.
+        """
+        owned_by_resources = list()
+        user_entity = self._driver.entity_unique_attribute(self.USER_TYPE, qualifiedName=user_id).entity
+
+        if not user_entity:
+            LOGGER.exception(f'User ({user_id}) not found in Atlas')
+            raise NotFoundException(f'User {user_id} not found.')
+
+        resource_guids = list()
+        for item in user_entity[self.REL_ATTRS_KEY].get('ownerOf') or list():
+            if (item['entityStatus'] == Status.ACTIVE and
+                    item['relationshipStatus'] == Status.ACTIVE and
+                    item['typeName'] == resource_type):
+                resource_guids.append(item[self.GUID_KEY])
+
+        entities = extract_entities(self._driver.entity_bulk(guid=resource_guids, ignoreRelationships=True))
+        if resource_type == self.TABLE_ENTITY:
+            owned_by_resources = self._serialize_popular_tables(entities)
+
+        return owned_by_resources
+
+    def get_dashboard_by_user_relation(self, *, user_email: str, relation_type: UserResourceRel) \
+            -> Dict[str, List[DashboardSummary]]:
+        pass
+
+    def get_table_by_user_relation(self, *, user_email: str, relation_type: UserResourceRel) -> Dict[str, Any]:
+        results = list()
```

Comment:
can we instantiate empty lists with `[]`?

Reply:
If any empty list or dict, in the python community we use list() or dict()
If there aren any elements during initialization, then use [] or {}

I'd leave it like that for now.